### ISSUE TITLE
Can read_excel() allow reading of open Excel workbooks/sheets?

### ISSUE THREAD

**Issue by mikedolanfliss** [tidyverse/readxl#732]
Like many, frequent user of this stellar package. Having perused issues and not finding anything:  

For my work flows it's not uncommon to occasionally have an excel file open, make edits, and then need to read it in as I iterate on a file. As is, I'm thrown errors that the excel file has to be closed for read permissions (even when using read_excel, not writing anything); might some integration of read_only open request under the hood allow reading of currently open xlsx/xls files, circumventing the permission error, instead of the close-open-close-open flow I have now?

**Comment by jennybc** [comment]
I assume you’re on Windows?

**Comment by mikedolanfliss** [comment]
Yes, apologies. I see this on Windows 10 and 11, read_excel not reading open files.

**Comment by jennybc** [comment]
I'm not in a position to do experiments on Windows right now, but I'm going to speculate that the answer is "no" there's not much that can be done, due to file locking behaviour on Windows. So, on the basis of that, I'm going to classify this as #wontfix (#cantfix, probably).
